package.json

```diff
@@ -24,6 +24,7 @@
     "test": "jest --coverage --no-cache",
     "build": "rollup -cm -n hyperapp -f umd -i src/index.js -o dist/hyperapp.js",
     "prepublish": "npm run build",
+    "flow": "./node_modules/.bin/flow",
```

Comment:
Please send another PR adding flow if you like, but let's keep this PR just for doc changes. Thanks! 👍